features/api/services/applications/index.feature

```diff
@@ -6,8 +6,8 @@ Feature: Product > Applications
 
   Background:
     Given a provider
-    And a product "My API"
-    And another product "Another API"
+    And a product "My API" with no plans
```

Comment:
Just to make sure that the plans created below are the only plans on the respective services.